src/message/socket-wrapper.js

```diff
@@ -40,6 +42,53 @@ var SocketWrapper = function( socket, options ) {
 };
 
 utils.inherits( SocketWrapper, EventEmitter );
+SocketWrapper.lastPreparedMessage = null;
```

Comment:
> Updates lastPreparedMessage and returns the [uws] prepared message.

And a uws prepared message is? Why is it static? What does it do? In nodeJS this concept is not as well known and since it calls into uws i would argue that it isn't obvious at all why its being used.

Same thing for finalizeMessage. Why is it done after sending the message? It is a flush? Is it a delete?